packages/account/src/Components/forms/form-fields.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Field } from 'formik';
+import { DateOfBirthPicker, Input } from '@deriv/components';
+import { toMoment } from '@deriv/shared';
+
+export const DateOfBirthField = props => (
+    <Field name={props.name}>
+        {({ field: { value }, form: { setFieldValue, errors, touched, setTouched } }) => (
+            <DateOfBirthPicker
+                error={touched.date_of_birth && errors.date_of_birth}
+                onBlur={() =>
+                    setTouched({
+                        ...touched,
```

Comment:
We can omit setting the previous touched values as it merges the new field value.